### PR TITLE
Initial frontend

### PR DESCRIPTION
**Pull request by mvdam**
* Frontend based on CRKA https://github.com/JetBrains/create-react-kotlin-app
* Kotlin -> React

### CODE REVIEW

File: frontend/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1 @@
+>1%
```
Comment (Bjeaurn) [inline frontend/.browserslistrc]: What is this exactly?

File: frontend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
```
Comment (Bjeaurn) [inline frontend/.gitignore]: .env* ?

File: frontend/src/search/Search.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    top: 30px;
+    left: 30px;
+    width: 300px;
+    background-color: white;
+    border-radius: 5px;
+    box-shadow: 0px 0px 17px 0px rgba(0,0,0,0.2);
+    padding: 10px;
+}
+
+.From, .To {
```
Comment (Bjeaurn) [inline frontend/src/search/Search.css]: Is it possible to use SCSS perhaps? Maybe for later in the project

Comment (mvdam) [reply frontend/src/search/Search.css]: Great suggestion; I've added it to the backlog!

File: frontend/.browserslistrc (hunk 1 above)
Comment (mvdam) [reply frontend/.browserslistrc]: I've added it to package.json.

File: frontend/.gitignore (hunk 2 above)
Comment (mvdam) [reply frontend/.gitignore]: Added it as well 😊

**Comment by Bjeaurn** [comment]
LGTM!

**Pull request merged**